pkg/cmd/admin/prune/images.go

```diff
@@ -99,6 +115,16 @@ func (o *PruneImagesOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command,
 		return kcmdutil.UsageError(cmd, "no arguments are allowed to this command")
 	}
 
+	if !cmd.Flags().Lookup("keep-younger-than").Changed {
```

Comment:
Confirmed, iow. [this will be true](https://github.com/openshift/origin/blob/ed89a4ef11531864926dcd603efa78555bd2b67b/Godeps/_workspace/src/github.com/spf13/pflag/flag.go#L158) if flag is set by a user, value does not matter:
```
$ oadm prune images --prune-over-size-limit=false --keep-younger-than=3h
error: --prune-over-size-limit cannot be specified with --keep-tag-revisions nor --keep-younger-than

$ oadm prune images --prune-over-size-limit=true --keep-younger-than=3h
error: --prune-over-size-limit cannot be specified with --keep-tag-revisions nor --keep-younger-than
```

Is that ok, or you want this to fail only when set to `true`?
```